api/v1/mongodbcommunity_types.go

```diff
@@ -523,8 +523,12 @@ func (m MongoDBCommunity) Hosts() []string {
 }
 
 // ServiceName returns the name of the Service that should be created for
-// this resource
+// this resource which is either the default name resourcename-svc or the user defined name
```

Comment:
[nit] I don't think implementation details necessarily need to be included in the docstring. The new text added is re-stating the body of the function. What do you think?

Reply:
agree